internal/reconciler/logpipeline/validator.go

```diff
@@ -33,6 +38,12 @@ func (v *Validator) validate(ctx context.Context, pipeline *telemetryv1alpha1.Lo
 		return err
 	}
 
+	if pipeline.Spec.Output.Custom == "" {
```

Comment:
Why not directly check for HTTP output?

Reply:
See: `test/e2e/logs_custom_output_test.go`

Reply:
I misunderstood what `custom` does, and omitted the `loki` output. Implementing your review comment...